Kudu.Services/Diagnostics/ProfileManager.cs

```diff
@@ -0,0 +1,314 @@
+﻿using Kudu.Contracts.Tracing;
```

Comment:
There may be some things you can't actually test, like starting the actual profile, but all the other logic in that class should be made testable in some form, so we can confidently make changes and have tests that allow us to step through the code, catch regressions, etc.